linera-views-derive/src/lib.rs

```diff
@@ -35,13 +38,78 @@ fn get_type_field(field: syn::Field) -> Option<syn::Ident> {
     }
 }
 
+fn crate_path(attributes: &[Attribute]) -> TokenStream2 {
+    attributes
+        .iter()
+        .find(|attribute| attribute.path.is_ident("crate_path"))
+        .map(|attribute| match attribute.parse_meta() {
+            Ok(Meta::NameValue(MetaNameValue {
+                lit: Lit::Str(value),
+                ..
+            })) => {
+                let path: Path = value.parse().expect("Invalid specific context");
+                path.into_token_stream()
+            }
+            _ => panic!(
+                r#"Invalid `crate_path` attribute syntax. \
+                Expected syntax: `#[crate_path = "path::to::linera_views"]`"#,
```

Comment:
Same comment: should it be `#[view(crate_path = "...")]` ?
Also, I couldn't find where this feature is actually used.

Reply:
Done.